README.md

```diff
@@ -184,6 +184,8 @@ the first available will be chosen, but you can force a specific one:
 let test#python#runner = 'pytest'
 " or
 let test#python#runner = 'nose'
+" or
+let test#python#runner = 'django'
```

Comment:
Ok. I've thought about it and talked to some people and I think you're right. I was worried about a case where you're testing your Django app with nose, but a) that's not the default or majority at all, and b) our nose runner wouldn't be able to handle that case (tests are still run through manage.py).

So, I think the safe thing to do is exactly what you're suggesting. We detect if you're a Django project, and if you are you get the Django test runner. And if people want to get fancy with alternate runners for Django we can deal with that in a separate PR.

I've made the change.